source/_addons/configurator.markdown

```diff
@@ -62,22 +62,73 @@ Screenshot of the HASS Configurator.
-- **ignore_pattern** (*Required*): Files and folders to ignore in the UI.
-- **dirsfirst** (*Required*): List directories before files in the file browser.
-- **enforce_basepath** (*Required*): If set to `true`, access is limited to files within the `/config` directory.
-- **notify_service** (*Required*): Specify a custom notify-service to be used to push notifications.
-- **ignore_ssl** (*Required*): Ignore SSL errors when accessing the Home Assistant API.
-- **sesame** (*Optional*): Secret token to dynamically allow access from the IP the request originates from. Open your bookmark https://hassio.yourdomain.com:8123/somesecretnobodycanguess while `allowed_networks` is set to `[]` and your IP will get whitelisted. You can use the _Network status_ menu to revoke IP addresses for which access has been granted. Regular authentication is still required.
-- **sesame_totp_secret** (*Optional*): Like the `sesame` option, but instead as Base32 encoded secret string must be provided. This string then can be added to a TOTP App like Google Authenticator. This way you get a 6-digit `sesame` that changes every 30 seconds.
-- **loglevel** (*Optional*): You can change the logging level from the default value `info` if you want to. Valid values are: `debug`, `info`, `warning`, `error`, `critical`.
+{% configuration %}
+username:
+  description: Set a username so that access your configuration is protected.
+  required: true
+  type: string
+password:
+  description: Set a password for access.
+  required: true
+  type: string
+ssl:
+  description: Enable or Disable SSL/TLS for the editor.
+  required: true
+  type: string
+certfile:
+  description: Set the path the your SSL certificate if the ssl-option is set to `true`.
+  required: true
+  type: string
+keyfile:
+  description: Set the path the your SSL private key if the ssl-option is set to `true`.
+  required: true
+  type: string
+allowed_networks:
+  description: Limit access to the configurator by adding allowed IP addresses/networks to the list.
+  required: true
+  type: string
+banned_ips:
+  description: List of statically banned IP addresses.
+  required: true
+  type: string
+banlimit:
+  description: Ban access from IPs after `banlimit` failed login attempts. This feature is disabled by default. Restart the add-on to clear the list of banned IP addresses.
+  required: true
+  type: int
+  default: 0
+ignore_pattern:
+  description: Regex of files and folders to ignore in the UI
+  required: true
+  type: string
+dirsfirst:
+  description: List directories before files in the file browser.
+  required: true
+  type: string
+enforce_basepath:
+  description: If set to `true`, access is limited to files within the `/config` directory.
+  required: true
+  type: string
+notify_service:
+  description: Specify a custom notify-service to be used to push notifications.
+  required: true
+  type: string
+ignore_ssl:
```

Comment:
`ignore_ssl` isn't used anymore. Please remove it from the example configuration as well.